tests/foreman/cli/test_lifecycleenvironment.py

```diff
@@ -392,11 +240,11 @@ def test_positive_list_with_pagination(self):
                 lces = LifecycleEnvironment.list(
```

Comment:
Now that we are parametrizing test we don't need `with self.subTest(per_page):`
so new test should look like:
```
        lces = LifecycleEnvironment.list(
            {'organization-id': self.org['id'], 'per-page': per_page}
        )
        assert len(lces) == per_page
        # Verify pagination and total amount of pages by checking the
        # items count on the last page
        last_page = ceil(self.lces_count / per_page)
        lces = LifecycleEnvironment.list(
            {'organization-id': self.org['id'], 'page': last_page, 'per-page': per_page}
        )
        assert len(lces) == self.lces_count % per_page or per_page
```